src/app/GitUI/UserControls/BlameControl.cs

```diff
@@ -522,23 +519,10 @@ private List<GitBlameEntry> CalculateBlameGutterData(IReadOnlyList<GitBlameLine>
 
         private void ActiveTextAreaControlDoubleClick(object sender, EventArgs e)
         {
-            if (_lastBlameLine is null)
-            {
-                return;
-            }
-
-            ObjectId selectedId = _lastBlameLine.Commit.ObjectId;
-            if (_revisionGridUpdate is null)
+            if (_lastBlameLine is not null
+                && TryGetRevision(_lastBlameLine.Commit, out (GitRevision SelectedRevision, string Filename) blameInfo))
             {
-                using FormCommitDiff frm = new(UICommands, selectedId);
-                frm.ShowDialog(this);
```

Comment:
popup removed, ok for me

Reply:
> popup removed, ok for me

This was duplicated code. The same as before is done in `BlameRevision` (at line 706/788), which is now called here, too.